### PR TITLE
Update .travis

### PR DESCRIPTION
**Pull request by ruscon**
As sample https://github.com/hwi/HWIOAuthBundle/blob/master/.travis.yml

### CODE REVIEW

**Comment by ruscon** [comment]
done

File: composer.json [hunk 1]
```diff
@@ -13,8 +13,12 @@
         }
     ],
     "require": {
-        "sonata-project/media-bundle": "~2.3"
+        "sonata-project/media-bundle": "~2.3|~3.0"
     },
+	"require-dev": {
```
Comment (dunglas) [inline composer.json]: There is an indentation problem here.

**Comment by dunglas** [comment]
Awesome work!

File: composer.json (hunk 1 above)
Comment (ruscon) [inline composer.json]: done

**Comment by dunglas** [comment]
Thank you @ruscon!

**Pull request merged**